htdocs/takepos/invoice.php

```diff
@@ -1143,6 +1143,8 @@ function SetNote() {
 	$s = $langs->trans("Customer");
 	if ($invoice->id > 0 && ($invoice->socid != getDolGlobalString($constforcompanyid))) {
 		$s = $soc->name;
+	} else {
+		print '$("#idcustomer").val("");';
```

Comment:
In most use cases for POS, the customer is a generic customer so a customer not modified, for such use case, removing selectedx customer seems to add a new step to select the customer if we reset it ?

Reply:
ok with you , i introduce now a new constant TAKEPOS_FORCE_CUSTOMER_SELECT

Reply:
When in default use case, when
`if ($invoice->id > 0 && ($invoice->socid != getDolGlobalString($constforcompanyid))) {`
was true, we did the print and then execut line 1275.
After your PR, code on line 1275 is no more executed for the default use case. Should be executed with default use case, like in old version, right ?

Reply:
ok @eldy i restore the default use case.